aospy/automate.py

```diff
@@ -295,19 +295,28 @@ def _exec_calcs(calcs, parallelize=False, client=None, **compute_kwargs):
         def func(calc):
             """Wrap _compute_or_skip_on_error to require only the calc
             argument"""
-            return _compute_or_skip_on_error(calc, compute_kwargs)
+            return _compute_or_skip_on_error(calc, {'write_to_tar': False})
 
         if client is None:
             n_workers = _n_workers_for_local_cluster(calcs)
             with distributed.LocalCluster(n_workers=n_workers) as cluster:
                 with distributed.Client(cluster) as client:
-                    return _submit_calcs_on_client(calcs, client, func)
+                    result = _submit_calcs_on_client(calcs, client, func)
         else:
-            return _submit_calcs_on_client(calcs, client, func)
+            result = _submit_calcs_on_client(calcs, client, func)
+        _serial_write_to_tar(calcs, **compute_kwargs)
```

Comment:
Similar to above comment: replace `**compute_kwargs` with `write_to_tar=compute_kwargs['write_to_tar']`.